### DISCUSSION TITLE
Welcome to AoTTG-2 Discussions!

### DISCUSSION THREAD

**Discussion by Lithrun** [AoTTG-2/AoTTG-2#305] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lithrun** [comment]
Hi Lithrun here! I presume most of you already know me due to my role as the one who started the AoTTG2 community, but also due to my active role as Lead Developer since March, so a short introduction.

I'm a software developer from the Netherlands that loves to investigate the latest innovations and technology, work with security, and write highly maintainable code. When I started with AoTTG2, my knowledge over Unity has been fairly limited, but after rewriting several critical components of the game, experimenting and studying Unity, my knowledge is more than good enough know.

I was introduced to the AoTTG community back in Late 2016, and created the famous "Expeditions Beyond the Walls" community in March 2017 where we introduced the Expeditions Gamemode into AoTTG. My main role within the community was being a server owner of the expeditions community until I started with AoTTG2.

For AoTTG2 as Lead Developer my roles are to Supervise the Managers, Supervise the Development Team, defining the roadmap. Besides those, I spend most of my time on developing AoTTG2. Not only on the game itself, but also on the Photon Server, API and Database. I'm an all-round developer as I'd like to say, so I just go wherever is needed. No task is too difficult for me, as I'm always up for a challenge! (But rewriting some classes with a cyclomatic complexity of +1000... if else else if else else if else else else else else else... stackoverflowexception...)

\- Lithrun

**Comment by rnetiks** [comment]
I-- am just gonna follow after Lithrun then.
Hey, Rnetiks / Space-Fox here, i first learned about AoTTG2 by pure accident when searching through the aottg reddit.
After 2 Days in the Discord server i decided i wanted to help the Project, although i got accepted by deceiving and saying i've been doing C# only for 2 Weeks, this was mainly due to the fact that i am a lazy person and Lith wont expect a newbie to do lots of stuff.
In actuality i've fiddled with C# since i was 11-12ish Years old, i am a perfectionist when it comes to graphical design and scalable UX, recently i've also been trying to get into Security and Maintainability.

My introduction to the game itself began 2013-2014ish, and the person that was the sole reason for me to stay left long ago.
I am from Germany, 20 Years old (born 19.11.1999) and always have an open ear for anyones problems, opinions or smalltalk.

My current Position within the Project is that of a Junior Developer, hopefully Developer in the Future.
now for the last information about me:
1. I am Bilingual (English & German), but i wanna increase the languages i know and recently started with French, after that i wanna do Japanese
2. 0 122 255 (If you understand it, kudos to you)
3. I Love Cats, so feel free to send me Cat and or Fox Images!
4. I know quite a few Coding Languages, sorted by how long i've been using them

> C# (8-9 Years)
> Lua (4 Years)
> Python (3 Years)
> PhP (2 1/4 Years)
> Javascript (2 Years)
> JQuery (1 Year)
> SQL (1 Year)
> C (1 Month)
> Brainfuck (1 Month)
> C++ (2 Days)

and thats the end of my introduction, best regards

- rne
